pkg/gofr/datasource/cassandra/interfaces.go

```diff
@@ -0,0 +1,36 @@
+package cassandra
+
+import "github.com/gocql/gocql"
+
+// clusterConfig defines methods for interacting with a Cassandra clusterConfig. This interface
```

Comment:
Instead of repeating this line multiple times:
`This interface is designed to be mockable for unit testing purposes, allowing you to control the behavior`

We can say it commonly like `All the interfaces are designed .......` and place it at the top